keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -331,14 +331,13 @@ metadata:
   serviceAccountName: [5]
   accessSecretName: [6]
   provisioner: [7]
-  policyActions [8]
-  parameters: [9]
+  parameters: [8]
```

Comment:
My understanding is that we want policyActions in the BA and it is an opaque, base-64 encoded string that reflects the configMap data values at the time of the binding.